content/docs/components/central-dashboard.md

```diff
@@ -0,0 +1,11 @@
++++
+title = "Central Dashboard"
+description = "All about central dashboard in the Kubeflow Ecosystem"
+weight = 10
++++
+
+Your Kubeflow deployment includes Central Dashboard as a key component to provide quick access to all components deployed in your cluster. Some features include:
+- Quick action, pipelines cards, metrics, and high level kubernetes events for an overview of your jobs and cluster in one view
```

Comment:
This list does not render properly. See the preview: https://deploy-preview-1538--competent-brattain-de2d6d.netlify.com/docs/central-dash/

To fix, add empty lines before and after lists.